worlds/dlcquest/data/items.csv

```diff
@@ -28,21 +28,22 @@ id,name,classification,groups
 26,DLC NPC Pack,progression,"DLC,Freemium"
 27,Cut Content Pack,progression,"DLC,Freemium"
 28,Name Change Pack,progression,"DLC,Freemium"
-29,Pickaxe,progression,"Item,Freemium"
 30,Season Pass,progression,"DLC,Freemium"
 31,High Definition Next Gen Pack,filler,"DLC,Freemium"
 32,Increased HP Pack,useful,"DLC,Freemium"
 33,Removed Ads Pack,filler,"DLC,Freemium"
 34,Big Sword Pack,progression,"DLC,Freemium"
 35,Really Big Sword Pack,progression,"DLC,Freemium"
 36,Unfathomable Sword Pack,progression,"DLC,Freemium"
-37,Gun,progression,"Item,DLCQuest"
-38,Sword,progression,"Item,DLCQuest"
-39,Wooden Sword,progression,"Item,Freemium"
+37,Progressive Weapon Basic,progression,"Item,Twice,DLCQuest"
```

Comment:
I would recommend naming these using the same pattern as the coin bundles
DLC Quest: Progressive Weapon
Live Freemium or Die: Progressive Weapon

Reply:
done